ChaosMod/Effects/db/Player/PlayerBlimpStrats.cpp

```diff
@@ -0,0 +1,31 @@
+/*
+	Effect by Last0xygen
+*/
+
+#include <stdafx.h>
+#include "Memory/Hooks/ScriptThreadRunHook.h"
+
+static void OnStart()
+{
+	Hash blimpHash = GET_HASH_KEY("blimp");
+	LoadModel(blimpHash);
+	Hooks::EnableScriptThreadBlock();
+	Vehicle veh = CreatePoolVehicle(blimpHash, -377.276, 1055.06, 340.962, 80);
```

Comment:
Shouldn't this be a normal vehicle which should be marked as despawnable afterwards instead?

Reply:
Yeah, would be a better idea :D